lib/wor/paginate/adapters/active_record.rb

```diff
@@ -15,10 +15,16 @@ def paginated_content
           @paginated_content ||= @content.offset(offset).limit(@limit)
         end
 
-        delegate :count, to: :paginated_content
+        def count
```

Comment:
This method should be public? Maybe should be private

Reply:
Nop. It is called from `Wor::Paginate::Formatter`